services/motion/client.go

```diff
@@ -136,10 +136,7 @@ func (c *client) MoveOnGlobe(
 	if len(obstacles) > 0 {
 		obstaclesProto := make([]*commonpb.GeoObstacle, 0, len(obstacles))
 		for _, eachObst := range obstacles {
-			convObst, err := spatialmath.GeoObstacleToProtobuf(eachObst)
-			if err != nil {
-				return false, err
-			}
+			convObst := spatialmath.GeoObstacleToProtobuf(eachObst)
 			obstaclesProto = append(obstaclesProto, convObst)
```

Comment:
Another nit is that `eachObst` is not a good name for this variable (which I guess I missed on the initial commit) since at any time through the loop it is only one instance of an `obstacle`.  An easy convention to follow is `for _, object := range objects`